src/app/content/utils/getCleanContent.ts

```diff
@@ -14,7 +16,10 @@ export default function getCleanContent(
 
   const pageContent = cachedPage ? cachedPage.content : '';
 
-  return transformer(pageContent)
+  const originalUrl = (book && page && archiveLoader.book(book.id, book.version).page(page.id).url())
+    || 'https://openstax.org/apps/archive/20201029.133542';
```

Comment:
this is what's making it work I think @TomWoodward

Reply:
this isn't a page url i'm surprised it would be working at all with this in here